src/js/events.js

```diff
@@ -21,6 +16,11 @@ import {
   isLocalStorageAvailable,
   isRunningOnWindows
 } from './helpers.js'
+import {
+  get_coprimes,
+  get_rank2_mode,
+  get_rational_approximations
+} from './sequences.js'
```

Comment:
I suggest creating a helpers folder and move all helper files into that. Consequently helpers.js would be renamed to something else, like `helpers/general.js`